### PR TITLE
Fix onward 404s across site

### PR DESCRIPTION
**Pull request by kelvin-chappell**
I don't really understand why this code is being called on non-hosted pages but this should fix it.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/hosted/onward.js [hunk 1]
```diff
@@ -10,16 +10,20 @@ define([
...
-            + 'onward.json')
-            .then(function (json) {
-                return fastdom.write(function () {
-                    placeholder.innerHTML = json.html;
+            var placeholder = document.querySelector('.js-onward-placeholder');
+
+            return fetchJson(config.page.ajaxUrl + '/'
+                + config.page.pageId + '/'
+                + config.page.contentType.toLowerCase() + '/'
+                + 'onward.json')
```
Comment (jfsoul) [inline static/src/javascripts/projects/commercial/modules/hosted/onward.js]: I might not be understanding the original PR but is this intended to hit the route `/advertiser-content/:campaignName/:pageName/:cType/onward.json` in commercial?

Comment (kelvin-chappell) [inline static/src/javascripts/projects/commercial/modules/hosted/onward.js]: Yes

Comment (jfsoul) [inline static/src/javascripts/projects/commercial/modules/hosted/onward.js]: is it missing the `/advertiser-content/:campaignName/` part or is that included in `pageId`?

**Comment by jfsoul** [comment]
👍  for getting this out to fix the 404s across the site

**Comment by NataliaLKB** [comment]
@kelvin-chappell merge straight away to fix the 404, but I think we should still deal with @gtrufitt 's question in the slack channel and @jfsoul 's question here.

**Comment by NataliaLKB** [comment]
Thanks @kelvin-chappell !

**Comment by CWCSociety** [comment]
[guardian/frontend] Fix onward 404s across site (#14526)


Inbox
x




Kelvin Chappell <notifications@github.com> 
7:38 AM (55 minutes ago)

Reply
to guardian/front. 






  This message is eligible for Automatic Cleanup! (notifications@github.com) Add cleanup rule | More info 

@kelvin-chappell commented on this pull request.

**Comment by CWCSociety** [comment]
Is this what you are talking about

**Pull request merged**